src/Core/src/Platform/Android/MauiSwipeView.cs

```diff
@@ -643,10 +643,18 @@ void LayoutSwipeItems(List<AView> childs)
 							break;
 					}
 
-					child.Measure(
+					if (item is ISwipeItemView)
```

Comment:
Both branches call `child.Measure`, only the `MeasureSpecMode` differs. Can simplify:

```
var widthMode = item is ISwipeItemView ? MeasureSpecMode.AtMost : MeasureSpecMode.Exactly;
var heightMode = item is ISwipeItemView ? MeasureSpecMode.AtMost : MeasureSpecMode.Exactly;

child.Measure(
    MeasureSpec.MakeMeasureSpec(swipeItemWidth, widthMode),
    MeasureSpec.MakeMeasureSpec(swipeItemHeight, heightMode));

```

Reply:
Could you include a comment explaining why use a different `MeasureSpecMode` between SwipeItem and SwipeItemView?

Reply:
yes , I have added a comment

Reply:
> Could you include a comment explaining why use a different `MeasureSpecMode` between SwipeItem and SwipeItemView?

yes , I have added a comment